### DISCUSSION TITLE
The black case for MiniDexed shown at the project's main page...

### DISCUSSION THREAD

**Discussion by JaFre2003** [probonopd/MiniDexed#740] [Q&A] [answered]
Hi everyone!
Just signed up for this brilliant project.
Kudos for the developers!

One starting question:
Couldn't help to notice that black case for MiniDexed shown at the project's main page.
Where can i find and  buy one?

**Comment by probonopd** [comment, marked as answer]
Hi @JaFre2003. That was an early 3dprinted prototype (barely working and held together with glue) of what eventually became the much better snap-fit [3d printable file](https://github.com/probonopd/MiniDexed/wiki/Hardware#3d-printable-file).

There is also a nice remix:
https://github.com/probonopd/MiniDexed/discussions/701

**Reply by JaFre2003** [reply]
Ok... nice things you can do with a 3D Printer...

**Comment by omersiar** [comment]
I got my inspiration from it.

![PXL_20241101_094653440](https://github.com/user-attachments/assets/65f99321-9c71-4bec-bf59-074d363b45ef)
![PXL_20241101_094635830](https://github.com/user-attachments/assets/293ceb7d-f5b3-41d3-bdcb-63525046cea1)

**Reply by JaFre2003** [reply]
Like this box. Is it a jewlery box? Where did you get it?

**Reply by omersiar** [reply]
https://www.duracell.co.uk/upload/2019/12/chargers_CEF22_UK_2000x2000_2-1024x1024.jpg
It is a duracell charger box
from thrift store for 1.99 ;)

**Reply by JaFre2003** [reply]
Hmm... Very nice for the cost.
